scripts/world_bank/health_nutrition_population/health_expenditure/expenditure_unittests.py

```diff
@@ -0,0 +1,46 @@
+import filecmp
```

Comment:
Can this be moved to a separate PR for expenditure import?
Let's keep only the population related scripts for this PR.